check_jar_mode_compilation.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
Are there any differences between this script and the approximate mode checking script other than the name of the output .json file? If not, I think it's much better to only have one script and make that a parameter.